ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -69,6 +66,19 @@ get_nnapi_hw_type (const gchar * str)
   return (index < 0) ? NNAPI_UNKNOWN : index;
 }
 
+/**
```

Comment:
Why have you moved this function definition?

Reply:
just for the code readability